SlicerSofa/SlicerSofa.py

```diff
@@ -70,8 +70,10 @@ def __init__(self) -> None:
         self._rootNode = None
         self._simulationRunning = False
         self._currentStep = 0
-        self._totaSteps = 0
+        self._totalSteps = 0
         self._dt = 0
+        self.initialgrid = []
```

Comment:
For now I would leave this to the derived classes. Two reasons:

  - Not all simulations necessarily will deal with unstructured grids (some may use surface meshes as polydata)
  - Some simulations may imply the use and update of more than one model.